### PR TITLE
Fix newline in mix project description

### PR DESCRIPTION
**Pull request by deniskulicek**

### CODE REVIEW

**Comment by deniskulicek** [comment]
@joshsmith also ready for review

File: mix.exs [hunk 1]
```diff
@@ -26,7 +26,7 @@ defmodule AnalyticsElixir.Mixfile do
       ],
       source_url: "https://github.com/joshsmith/segment_elixir/",
       test_coverage: [tool: ExCoveralls],
-      version: "2.0.0-rc.1",
+      version: "2.0.0-rc.3",
```
Comment (joshsmith) [inline mix.exs]: Did this bump from `1` to `3`?

**Review by joshsmith — CHANGES_REQUESTED** [review]

File: mix.exs (hunk 1 above)
Comment (deniskulicek) [reply mix.exs]: there was a short-lived `rc.2` in between, here: https://github.com/deniskulicek/segment_elixir/releases, there will be some more PRs so it doesn't really matter right now

Comment (joshsmith) [reply mix.exs]: In that case, can we change it to `-rc.2` and rebase then? I haven't even published these to Hex past `-rc.1`, so I'd prefer we hold off until we're ready for another RC.

Comment (joshsmith) [reply mix.exs]: In fact, this doesn't have anything past `-rc.1` in this repository, so we can still correct back to there, right?

Comment (deniskulicek) [reply mix.exs]: yes, let's keep it as `-rc.1` until we have something that will go to hex

Comment (deniskulicek) [inline mix.exs]:
```suggestion
      version: "2.0.0-rc.1",
```

**Review by joshsmith — APPROVED** [review]

**Pull request merged**